src/transformers/commands/pt_to_tf.py

```diff
@@ -19,7 +19,8 @@
 import numpy as np
 from datasets import load_dataset
 
-from huggingface_hub import Repository, upload_file
+from huggingface_hub import Repository, create_commit
+from huggingface_hub._commit_api import CommitOperationAdd
```

Comment:
You'll need to gate those imports behind a version check: Transformers accepts any version of `huggingface_hub` from 0.1.0 and onward.